tests/test_modifier.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+from experimenter import modify_generator, queue, exceptions, utils
+from query import query_on_seeds
+
+class ModifierTestCase(unittest.TestCase):
+    
+    def test_seed_modifier(self):
+        #initialize the modifier with random-seed and a given max jobs
+        args = {'seed_limit':25, 'submitter':None, 'pipeline_id':None}
+        num_test = 21
+        modifier = modify_generator.ModifyGenerator('random-seed', num_test, {'seed_limit':25})
+        #start the counter to make sure there are the right amount of jobs
+        counter = 0
+        seed_old = 12.1
+        #begin the test if number of jobs is correct
+        for job in modifier:
+           counter += 1
+           _,_,seed_new = job
+           self.assertNotEqual(seed_old, seed_new)
+           seed_old = seed_new
+        self.assertEqual(counter,num_test) 
+        
+    def test_query_seeds(self):
+        args = {'seed_limit':25, 'submitter':'byu', 'pipeline_id':None}
+        query_results = query_on_seeds(args.pipeline_id, args.seed_limit, args.submitter)
```

Comment:
It looks like you did not run these tests. This is going to fail. Dictionary items cannot be accessed with the dot syntax.

Reply:
How do we run the tests found in the tests folder? Is there some way that we are supposed to run these besides calling them directly. (python3 -m experimenter tests/test_modifier.py)

Reply:
From the root of the repo, `python3 run_tests.py` will run all tests under `/tests`. To run just the tests in one file, you can use `python3 tests/<test file>`. To do this, you need to make sure that your test file has the right main function defined at the bottom of the file:

```
if __name__ == '__main__':
    unittest.main()
```